runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -182,16 +185,26 @@ public void keyPressed(KeyEvent e)
 					.toString()
 					.trim();
 
-				// 0 is username, 1 is password
-				if (client.getCurrentLoginField() == 0)
+				switch (client.getLoginIndex())
 				{
-					// Truncate data to maximum username length if necessary
-					client.setUsername(data.substring(0, Math.min(data.length(), MAX_USERNAME_LENGTH)));
-				}
-				else
-				{
-					// Truncate data to maximum password length if necessary
-					client.setPassword(data.substring(0, Math.min(data.length(), MAX_PASSWORD_LENGTH)));
+					case 2:
+						// 0 is username, 1 is password
+						if (client.getCurrentLoginField() == 0)
```

Comment:
I know this was added before, but same here. Documenting these values would be nice.

Reply:
I think at that point it should be just changed to enum.I just didnt bothered with it originally because the PR was stuck for so long

Reply:
Actually, its just 2 values, and its already documented. But I will update loginIndex